source/monitoring/cluster-monitoring.txt

```diff
@@ -81,53 +83,79 @@ outputs messages such as the following:
+provides the name of each SDAM event, linked to the class's API
+documentation, and a description of when the event is published:
 
 .. list-table::
    :widths: 35 65 
    :header-rows: 1
 
-   * - Event Name
+   * - Event Type
      - Description
 
-   * - ``ServerChangedEvent``
-     - Created when an instance state changes, such as from secondary to
-       primary.
+   * - :php:`ServerChangedEvent <mongodb-driver-monitoring-serverchangedevent>`
+     - Created when the server description changes, such as the server's type
+       changing from secondary to primary.
 
-   * - ``ServerOpeningEvent``
-     - Created when the server is initialized.
+   * - :php:`ServerOpeningEvent <mongodb-driver-monitoring-serveropeningevent>`
+     - Created when the server description is instantiated with its
+       defaults.
 
-   * - ``ServerClosedEvent``
+   * - :php:`ServerClosedEvent <mongodb-driver-monitoring-serverclosedevent>`
      - Created when the server is closed.
 
-   * - ``TopologyChangedEvent``
-     - Created when the topology changes, such as an election of a new
-       primary or disconnection of a ``mongos`` proxy.
+   * - :php:`TopologyChangedEvent <mongodb-driver-monitoring-topologychangedevent>`
+     - Created when the topology description changes, such when there is
+       an election of a new primary or disconnection of a ``mongos`` proxy.
 
-   * - ``TopologyOpeningEvent``
-     - Created when the topology is initialized.
+   * - :php:`TopologyOpeningEvent <mongodb-driver-monitoring-topologyopeningevent>`
+     - Created when the topology description is initialized.
 
-   * - ``TopologyClosedEvent``
+   * - :php:`TopologyClosedEvent <mongodb-driver-monitoring-topologyclosedevent>`
      - Created when the topology is closed.
 
-   * - ``ServerHeartbeatStartedEvent``
-     - Created when the heartbeat is started.
+   * - :php:`ServerHeartbeatStartedEvent <mongodb-driver-monitoring-serverheartbeatstartedevent>`
+     - Created when the server monitor sends a ``hello`` call to the server.
+       This action is called a heartbeat.
 
-   * - ``ServerHeartbeatSucceededEvent``
+   * - :php:`ServerHeartbeatSucceededEvent <mongodb-driver-monitoring-serverheartbeatsucceededevent>`
      - Created when the heartbeat succeeds.
 
-   * - ``ServerHeartbeatFailedEvent``
+   * - :php:`ServerHeartbeatFailedEvent <mongodb-driver-monitoring-serverheartbeatfailedevent>`
      - Created when the heartbeat fails.
 
+You can find a list of the monitoring subscriber classes and event
+methods in the :php:`Monitoring classes and subscriber functions
+<mongodb.monitoring>` section of the PHP manual.
+
+Remove a Subscriber
+-------------------
+
+Later in your application, you might not want to subscribe to
+SDAM events. To unregister a subscriber from your client, use the
+``MongoDB\Client::removeSubscriber()`` method. If you attempt to remove
+a nonexistent subscriber, the method doesn't perform any action.
```

Comment:
On a related note, PHPC (extension) also doesn't allow you to register the same subscriber twice. Attempting to do so would also be a no-op. I don't think we need to mention that, though.